receiver/cloudfoundryreceiver/README.md

```diff
@@ -0,0 +1,88 @@
+# Cloud Foundry Receiver
+
+The Cloud Foundry receiver connects to the RLP (Reverse Log Proxy) Gateway of the Cloud Foundry installation, typically
+available at the URL `https://log-stream.<cf-system-domain>`.
+
+## Authentication
+
+RLP Gateway authentication is performed by adding the Oauth2 token as the `Authorization` header. To obtain an OAuth2
+token to use for the RLP Gateway, a request is made to the UAA component which acts as the OAuth2 provider (URL
+specified by `uaa_url` configuration option, which typically is `https://uaa.<cf-system-domain>`). To authenticate with
+UAA, username and password/secret combination is used (`uaa_username` and `uaa_password` configuration options). This
+UAA user must have the `client_credentials` and `refresh_token` authorized grant types, and `logs.admin` authority.
+
+The following is an example sequence of commands to create the UAA user using the `uaac` command line utility:
+* `uaac target https://uaa.<cf-system-domain>`
+* `uaac token client get identity -s <identity-user-secret>`
+* `uaac client add <uaa_username> --name opentelemetry --secret <uaa_password> --authorized_grant_types client_credentials,refresh_token --authorities logs.admin`
+
+The `<uaa_username>` and `<uaa_password>` above can be set to anything as long as they match the values provided to the
+receiver configuration. The admin account (which is `identity` here) which has the permissions to create new clients may
+have a different name on different setups. The value of `--name` is not used for receiver configuration.
+
+## Configuration
+
+The receiver takes the following configuration options:
+* `rlp_gateway_url` - URL of the RLP gateway, required, typically `https://log-stream.<cf-system-domain>`
+* `rlp_gateway_skip_tls_verify` - whether to skip TLS verify for the RLP Gateway endpoint, default `false`
+* `rlp_gateway_shard_id` - metrics are load balanced among receivers that use the same shard ID, therefore this must 
+  only be set if there are multiple receivers which must both receive all the metrics instead of them being balanced
+  between them. Default value is `opentelemetry`
+* `uaa_url` - URL of the UAA provider, required, typically `https://uaa.<cf-system-domain>`
+* `uaa_skip_tls_verify` - whether to skip TLS verify for the UAA endpoint, default `false`
+* `uaa_username` - name of the UAA user (required grant types/authorities described above)
+* `uaa_username` - password of the UAA user
+* `http_timeout` - HTTP socket timeout used for RLP Gateway, default `10s`
```

Comment:
Can you format it to make it more readable? It should be easy to find if an option is required and what is its default value.

E.g. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#configuration

Reply:
Done.

Reply:
please resolve